kmir/src/kmir/convert_from_definition/__main__.py

```diff
@@ -19,7 +19,9 @@ def main() -> None:
     tools = semantics()
     p = Parser(tools.definition)
 
-    result = p.parse_mir_json(json.loads(args.json), args.sort)
+    with open(args.json, 'r') as f:
+        json_data = json.load(f)
+    result = p.parse_mir_json(json_data, args.sort)
```

Comment:
Working with files is imperative as soon as we get to reasonable program sizes... we can always use `/dev/stdin` otherwise. 
My workflow to debug/test this was 
```
$ jj -p some.path.#.into.the..1.json.structure < input.json | \
    poetry run -C kmir -- convert-from-definition /dev/stdin SomeSort
```
We could set up tests that way, too.